### PR TITLE
Mail Buff

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Mail Spawn 7 times less, worth 10 times more.
Mail now take 2 storage and not 4, and is now a small item size.
Icon of job is a bit bigger on mail.
Added back the mention of station income when you open mail.

## Why / Balance
Buff mail income. lower entity spam.

## How to test
Spawn in.
Use command Mailnow
Turn on the mail teleporter 
Open / pickup mail

## Media
N/A

## Breaking changes
N/A

**Changelog**
:cl: dvir01
- tweak: Frontier has increased mail related profits, but lowered the amount of existing mail, expect the mail to find you soon.

### CODE REVIEW

**Comment by whatston3** [comment]
Interesting.  Dividing the total mail by 7 sounds good, but seems maybe a bit too far?  I've got a mail table rework in progress - I don't know if I could find a consistent set of satisfying 35-minute worthy deliveries.  Will give this a closer look.

**Review by whatston3 — APPROVED** [review]
Looks good, works well under local tests.  Might suggest a shorter time between deliveries, but could be best to work that out through playtesting.

priorityDuration increase is good, especially for NFSD or science ship deliveries, where you might not have IFF signatures or where you need to make deep space deliveries.  Larger job stamp is great.  I like the reduced mail size, makes organizing larger mail runs easier.  Bringing the bounty back to messages provides feedback to openers that they're doing a good thing for the station.  Lovely stuff.

For the rates, this change helps mail carriers move towards making smaller deliveries and staying out longer - all good.  It incentivizes interactions between mail carrier and other ships' crew, and doesn't penalize the carrier for making longer courier runs.  With 200 letters to deliver max (math in another comment), there may be idle periods (still better than being backed up nonstop), and you're more likely to have people receiving few or no packages.

**Comment by whatston3** [comment]
For the change in rates - just running some quick numbers, in a 6 hour shift, you'll have 11 mail spawn opportunities, with about 8-18 letters per delivery (assuming 32-72 players).  Assuming the mail carrier works the full shift and doesn't miss a teleport, they'd be tasked with delivering about 200 letters maximum (currently about 1400), and 90ish on low pop hours - resulting in ~1.5MM and ~675k spesos in minimum possible bounties.

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/base_mail.yml [hunk 1]
```diff
@@ -19,7 +20,7 @@
       map: ["enum.MailVisualLayers.FragileStamp"]
       visible: false
     - map: ["enum.MailVisualLayers.JobStamp"]
-      scale: 0.5, 0.5
+      scale: 0.8, 0.8 # Frontier 0.5<0.8
       offset: 0.275, 0.2
```
[the remark this answers is not in the document]
Comment (dvir001) [reply Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/base_mail.yml]: Nice

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**